### DISCUSSION TITLE
迢递 | /cn/2022/01/miracle/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1625] [Comments]
稻场上
一场齐膝深的大雪
白花花
[&hellip;] 大雪底
一段料不见的造化
出发发
[&hellip;] 门缝边
一只五岁半的小伢
呆瓜瓜
[&hellip;] 百里外
一粒负三岁的种子
没芽芽
[&hellip;] &hellip;&hellip;
[&hellip;] 多年后
一句不经意的雪话
梦家家
[&hellip;] 今门前
一场齐膝深的大雪
路洼洼
[&hellip;] 天涯下
 &hellip;

https://yihui.org/cn/2022/01/miracle/

**Comment by think-crow** [comment]
新年好，益辉大哥。
初始之年，万事顺遂！

**Reply by yihui** [reply]
谢谢！
